lib/ex_prosemirror/type.ex

```diff
@@ -16,6 +16,19 @@ defmodule ExProsemirror.Type do
     struct_or_changeset
     |> cast(attrs, [])
     |> cast_prosemirror_content(opts)
+    |> maybe_force_inline(opts[:inline])
+  end
+
+  defp maybe_force_inline(changeset, false), do: changeset
+
+  defp maybe_force_inline(changeset, true) do
+    update_change(changeset, :content, fn content ->
+      if content = List.first(content) do
+        [content]
+      else
+        []
+      end
```

Comment:
could this be one lined ?

Reply:
I don't think :/ (yeah technically but for readibility)

Reply:
I update to

```elixir
case content do
  [content | _] -> content
  _ -> []
end
```

That seem a better solution